src/addScore.js

```diff
@@ -0,0 +1,19 @@
+async function addScore(data) {
```

Comment:
- [ ] Please, use arrow function according to ES6 structure in place function word, this will help to make your code with better practices. 🐶